view_distillation/code_used_for_paper/utils.py

```diff
@@ -0,0 +1,399 @@
+import random
```

Comment:
this utils.py is in a folder "code_used_for_paper" -- can you refactor this asap?